ClosedXML/Excel/CalcEngine/Functions/MathTrig.cs

```diff
@@ -927,8 +927,17 @@ private static object SqrtPi(List<Expression> p)
 
         private static object Subtotal(List<Expression> p)
         {
+            var expressions = p.Skip(1)
+                .SelectMany(e =>
+                    e is XObjectExpression xoe && xoe.Value is CellRangeReference crr
+                        ? crr.Range.CellsUsed()
+                            .Where(z => !z.FormulaA1?.StartsWith("SUBTOTAL(") ?? true) // except cells with the formula "SUBTOTAL"
```

Comment:
It is not that easy, unfortunately. 
1. String comparison must be case insensitive.
2. A formula may contain `SUBTOTAL` not only in the beginning but anywhere, for a cell to be excluded from totals. But `string.Contains` would also exclude cells having `SUBTOTALS` included as text (e,g, `=IF(A1="SUBTOTAL()", 10, 50)`). I admit this is an extremely rare case though.

Reply:
Now `SUBTOTAL` is searched using regex anywhere in the formula with case-insensitive. If a substring is found in the formula's text part, it will be ignored.